docs/semgrep-cloud-platform/getting-started.md

```diff
@@ -94,6 +76,95 @@ Using Semgrep Supply Chain with Semgrep Cloud Platform provides users with the V
+2. Check that the organization under the **Join an organization** button is the correct org to join. If there is more than 1 org, select the org to join.
+![DESCRIPTION](/img/join-an-org.png#md-width)
+3. Click **Join an organization**.
+
+## Running scans
+
+### Starting a SAST and SCA scan on a remote repository
+
+<PlatformAddRepo />
+
+#### Detecting GitHub repositories
+
+<PlatformDetectGhRepos />
+
+:::tip Product-specific information
+* To learn more about SAST scans on your codebase, see [Getting started with Semgrep Code](/semgrep-code/getting-started).
+* To learn more about SCA scans for your third-party dependencies, see [Getting started with Semgrep Supply Chain](/semgrep-supply-chain/getting-started).
+* Both products are **free for up to 10 contributors**. See [Usage limits](/usage-limits) to learn more about contributors and usage limits.
+:::
+
+### Starting a local repository scan and sending findings to SCP
+
+You can send scan results from a local repository to Semgrep Cloud Platform. The local repository is a separate **Project** from its remote counterpart.
+
+:::caution
+For team members of an organization, it is **not** recommended to send local repository scan results to the organization account. When logging in to Semgrep from the CLI through `semgrep login`, ensure that you are signed in SCP as your **personal account**. See [Projects separation between local and remote repositories](#projects-separation-between-local-and-remote-repositories).
+:::
+
+To send findings from a local repository, perform the following steps:
+
+1. Ensure that you are signed into Semgrep Cloud Platform in the account you want to send findings to. It is recommended to send local repository findings to your **personal** account.
+2. Log in to Semgrep:
+```
+semgrep login
+```
+2. Click the login URL provided. Your are taken to your web browser to complete the login process.
+3. Follow any additional steps.
+4. After logging in, start a scan in your CLI:
+```
+semgrep ci
+```
+
+#### Projects separation between local and remote repositories
+
+The Project slug for a remote repository always takes the form `account-name/repository-name`. The Project slug for a local repository always takes the form `repository-name`. Refer to the following screenshot for an example of both remote and local Projects in a single personal account.
+
+* **For personal accounts:** A local repository scan **never** overwrites findings records of its remote counterpart. They are two separate Projects. Personal accounts only have one team member or user: you.
+* For **organization accounts**: A local repository scan never overwrites findings records of its remote counterpart, but if two members both send local repository findings, their records can overwrite each other's findings. This is because org accounts can have more than one team member, but all local scans are sent to the same Project slug.

```

Comment:
```suggestion
* For **organization accounts**: A local repository scan does not overwrite the findings records of its remote counterpart, but if two members both send local repository findings, their records can overwrite each other's findings. This is because org accounts can have more than one team member, but all local scans are sent to the same Project slug.
```
"their records can overwrite each other's findings" sounds weird here but I can't think of a great replacement. If you can, though, I'm for using it :)

Reply:
You're right, it's awkward. 

Alternative: if two members both send local repository findings, one set of results may overwrite the other unintentionally.